Updated to Cordova 1.9.

wcwynn opened the pull request:
- Added example and updated the README.
- Added badge set/get methods.
- Added launch method.
- Added example code for optionally handling notifications in JS code.

Reviewer: @aogilvie

Commit:
Updated to Cordova 1.9.

- Added example and updated the README.
- Added badge set/get methods.
- Added launch method.
- Added example code for optionally handling notifications in JS code.

aogilvie commented on README.md:
```diff
@@ -4,13 +4,13 @@
 # PLUGIN: 
 
```
Update readme to reflect changes.

wcwynn replied:
Done.

aogilvie commented on example/www/index.html:
```diff
@@ -0,0 +1,171 @@
+    };
+	document.addEventListener("touchmove", preventBehavior, false);
+	*/
+	
```
add new "receivedLocalNotification" event if the developer want to get all local notifications.

wcwynn replied:
Done.

Commit:
Modifed plugin and example to reflect peer review.

- Modified the plugin to fire a notification event to inform JS about a
  triggered notification.

- Modified the handling of startup notifications to make it so that the JS
  developer does not need to be concerned with whether or not a notification was
  triggered at application launch time or not.

- Modified the required app delegate code required to listen for notificaiton
  events -- the required app delegate code is now simpler.

- Updated the README.md and example code to reflect these changes.

wcwynn commented:
I updated the pull request.  I think things should be a bit cleaner/simpler now but if there are other ideas how to make cleaner let me know.

Unfortunately, iOS doesn't have a built-in UIApplication<XXX>Notification for observing when a local/remote notification occurs so we can't remove everything from the app delegate.  I did try to trim it down to the bare minimum required code.  I suspect there is such a notification internally in iOS, but it seems Apple has not made it a public one yet.  If you know of a way to get rid of the app delegate code, let me know and I'll remove it.

Commit:
Update the example to include cordova framework.

aogilvie commented:
Awesome!
